tests/test_audio_ops_eager.py

```diff
@@ -68,6 +68,26 @@ def fixture_decode_wav():
 
   return args, func, expected
 
+@pytest.fixture(name="encode_wav", scope="module")
+def fixture_encode_wav():
+  """fixture_encode_wav"""
+  wav_path = os.path.join(
+      os.path.dirname(os.path.abspath(__file__)),
+      "test_audio", "ZASFX_ADSR_no_sustain.wav")
+  value = tf.io.read_file(wav_path)
+  path = os.path.join(
+      os.path.dirname(os.path.abspath(__file__)),
+      "test_audio", "ZASFX_ADSR_no_sustain.s16le.pcm")
+  audio = np.fromfile(path, np.int16)
+  audio = np.reshape(audio, [14336, 2])
+  audio = tf.convert_to_tensor(audio)
+
+  args = audio
+  func = lambda e: tfio.experimental.audio.encode_wav(e, rate=44100)
+  expected = value
```

Comment:
Maybe just renaming `value` to be `expected`?

Reply:
@terrytangyuan Ah didn't catch this one, will carry the update in the next PR.